packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -129,12 +114,13 @@ $ds-c-inset-border-width: $spacer-half;
 }
 
 // Display a checkmark
+// Using CSS to display the checkmark because background images don't how in high contrast mode on windows.
 .ds-c-choice:checked + label {
-  &::after {
-    opacity: 1;
-  }
   &::before {
     background-color: $color-primary;
+    background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 216 146'%3E%3Cpath fill='%23fff' d='M168.86 37.966l-11.08-11.08c-1.52-1.52-3.367-2.28-5.54-2.28-2.172 0-4.02.76-5.54 2.28L93.254 80.414 69.3 56.38c-1.52-1.522-3.367-2.282-5.54-2.282-2.172 0-4.02.76-5.54 2.28L47.14 67.46c-1.52 1.522-2.28 3.37-2.28 5.542 0 2.172.76 4.02 2.28 5.54l29.493 29.493 11.08 11.08c1.52 1.52 3.368 2.28 5.54 2.28 2.173 0 4.02-.76 5.54-2.28l11.082-11.08L168.86 49.05c1.52-1.52 2.283-3.37 2.283-5.54 0-2.174-.76-4.02-2.28-5.54z'/%3E%3C/svg%3E");
```

Comment:
Isnt this still a `background-image`? I thought that thats what caused the original issue.

Reply:
Yeah you're right I was thinking it might work differently but it doesn't. I think I'm going to go back to the full SCSS check without the rounded corners and update the check icon to match